### DISCUSSION TITLE
why pimcore field collection name is not allowing alphanumeric characters ?

### DISCUSSION THREAD

**Discussion by Karthi96** [pimcore/pimcore#9685] [Q&A / Support]
Hi Team, 

I'd like to create a field collection for different versions of products like product1,product2, ..etc. When we create a class it accepts numeric also but in field collection, it's not possible. is there any way where we can field collection with alphanumeric?

Thanks 
Karthikeyan B

**Comment by Karthi96** [comment]
any idea, how we can make an FC with the alphanumeric name?

**Reply by weisswurstkanone** [reply]
I think there is no real reason. Same rules as for classes could be applied I guess.
